tests/Functional/Command/SynchronizePackageCommandTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Tests\Functional\Command;
+
+use Buddy\Repman\Command\SynchronizePackageCommand;
+use Buddy\Repman\Message\Security\ScanPackage;
+use Buddy\Repman\Tests\Functional\FunctionalTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Messenger\Transport\InMemoryTransport;
+
+final class SynchronizePackageCommandTest extends FunctionalTestCase
+{
+    private string $userId;
+    private string $buddyId;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->userId = $this->createAndLoginAdmin();
+        $this->buddyId = $this->fixtures->createOrganization('buddy', $this->userId);
+    }
+
+    public function testSynchronizeSuccess(): void
+    {
+        $packageId = $this->fixtures->addPackage($this->buddyId, 'https://buddy.com');
+        $this->fixtures->syncPackageWithData($packageId, 'buddy-works/buddy', 'Test', '1.1.1', new \DateTimeImmutable());
+
+        $commandTester = new CommandTester($this->container()->get(SynchronizePackageCommand::class));
+        $result = $commandTester->execute([
+            'package id' => $packageId,
+        ]);
+
+        /** @var InMemoryTransport $transport */
+        $transport = $this->container()->get('messenger.transport.async');
+
+        self::assertCount(2, $transport->getSent());
```

Comment:
why two?

Reply:
there is `$this->fixtures->addPackage()` before, but I've added `$transport->reset()` now so we are checking only what command is doing